docs/api/security/psa.md

```diff
@@ -0,0 +1,38 @@
+## Mbed PSA
+
+### Terms and Abbreviations
+
+| Abbreviation | Description                         |
+|--------------|-------------------------------------|
+| PSA          | Platform Security Architecture      |
+| SPM          | Secure Partition Manager            |
+| SPE          | Secure Processing Environment       |
+| NSPE         | Non-Secure Processing Environment   |
+| IPC          | Inter Process Communication         |
+| RoT          | Root Of Trust                       |
+
+
+### Overview
+Mbed PSA provides essential root of trust services and infrastructure for developing robust IoT applications.
+When mbed-os is running on PSA Security Model compliant target, Mbed PSA helps to protect cryptographic assets, credentials, and critical code sections by providing an isolation between a Secure Processing Environment (SPE) and a Non-Secure Processing Environment (NSPE). The isolation is managed by Secure Partition Manager (SPM) which utilizes unique HW features available on the target. SPM provides standardized IPC APIs which abstract the fact that partitions could be living inside a virtualized environment (v8M, TEE on Cortex-A), or another chip.
```

Comment:
I think we should use Mbed OS (not mbed os)

Reply:
done